docs/csharp/index.yml

```diff
@@ -0,0 +1,113 @@
+### YamlMime:Landing
+
+title: "C# documentation"
+summary: "Learn how to write any application using the C# programming language on the .NET platform."
+
+metadata:
+  title: "C# docs - get started, tutorials, reference."
+  description: "Learn C# programming - for beginning developers, developers new to C#, and experienced C# / .NET developers"
+  ms.topic: landing-page # Required
+  author: billwagner
+  ms.author: wiwagn
+  ms.date: 11/19/2019
+
+landingContent:
+  - title: "Learn to program in C#"
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: Online, interactive Hello world
+            url: tutorials/intro-to-csharp/hello-world.yml
+          - text: "Math and numbers in C#"
+            url: tutorials/intro-to-csharp/numbers-in-csharp.yml
+          - text: Introductory tutorials
+            url: tutorials/intro-to-csharp/index.md
+          - text: "Create C# apps with Visual Studio"
+            url: https://docs.microsoft.com/visualstudio/get-started/csharp/?view=vs-2019
+      - linkListType: video
+        links:
+          - text: "What is C#?"
+            url: https://channel9.msdn.com/Series/CSharp-101/What-is-C/player
+          - text: "C# 101 video series"
+            url: https://aka.ms/dotnet3-csharp
+      - linkListType: download
+        links:
+          - text: Download the .NET Core SDK
+            url: https://dotnet.microsoft.com/download
+
+  - title: "C# fundamentals"
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: "A tour of C#"
+            url: tour-of-csharp/index.md
+          - text: "Inside a C# program"
+            url: programming-guide/inside-a-program/index.md
+          - text: Main() and command-line arguments
+            url: programming-guide/main-and-command-args/index.md
+      - linkListType: concept
+        links:
+          - text: "The C# and .NET type system"
+            url: programming-guide/types/index.md
+          - text: Language Integrated Query (LINQ)
+            url: linq/index.md
+          - text: async and await
+            url: programming-guide/concepts/async/index.md
+      - linkListType: tutorial
+        links:
+          - text: Object-oriented inheritance
+            url: tutorials/inheritance.md
+          - text: Use LINQ to query data
+            url: tutorials/working-with-linq.md
+          - text: Format text using string interpolation
+            url: tutorials/string-interpolation.md
+          - text: Fundamental language tutorials
```

Comment:
I wouldn't expect to see the tutorials homepage from this title.

Get started C# tutorials maybe?